dali/kernels/reduce/reduce_drop_dims.h

```diff
@@ -51,7 +51,7 @@ struct DropDims {
   int64_t mod[kMaxDims];
   int start = 2 * kMaxDims;
 
-  DropDims() = default;
+  DALI_HOST_DEV DropDims() {}
```

Comment:
NVCC9 is produces error with =default when there's no host/device and a warning when it is there :(